frontend/src/components/PenRequestDetail.vue

```diff
@@ -403,7 +403,10 @@
                     </v-col>
                   </v-row>
                   <v-row justify="end" class="px-3">
-                    <v-col cols="12" xl="3" lg="5" md="5" class="pt-0">
+                    <v-col justify="right" style="text-align: right;" cols="8" xl="3" lg="5" md="5" class="pt-0">
+                      <span  v-if="this.numberOfDuplicatePenRequests > 0"><span style="color: red;font-weight: bold;font-size: 1.2rem;">{{this.numberOfDuplicatePenRequests}}</span><span style="color: red;font-weight: normal;font-size: 1rem;"> prior PEN Requests</span></span>
```

Comment:
Vuetify has classes that you can use for styles: font-weight-bold red--text

Reply:
example please